modules/ROOT/pages/cbl-whatsnew.adoc

```diff
@@ -17,6 +17,27 @@ NOTE: Couchbase Lite 3.0 introduces some breaking changes. +
 If you are upgrading from 2.x, please refer to the appropriate upgrade page -- see: <<lbl-upgrade>>
 Users should be able to upgrade to 3.1.x from 3.0.x without manual intervention.
 
+== Release 3.2.0 - Beta 3 - C (June 2024)
```

Comment:
Just to check I understand what you're doing here: all this info is in the whatsnew during beta, and referred to from the minimal relnotes.

After beta, will that be folded back into the GA relnote?

Reply:
Most likely I'll fold them into a general description of Vector Search plus the API additions yeah.